src/game.js

```diff
@@ -0,0 +1,83 @@
+export class Car {
+  #name;
+  #score;
+  constructor(name) {
+    this.#name = name;
+    this.#score = '';
+  }
+  get name() {
+    return this.#name;
+  }
+  get score() {
+    return this.#score;
+  }
+  set score(score) {
+    this.#score = score;
+  }
+}
+
+export class RacingCarGame {
+  constructor() {
+    this._cars = [];
+    this._racingCount = 0;
+    this.message = '';
+  }
+
+  get cars() {
+    return this._cars;
+  }
+
+  set cars(carList) {
+    this._cars = carList;
+  }
+
+  set racingCount(racingCount) {
+    this._racingCount = racingCount;
+  }
+
+  play() {
+    for (let i = 0; i < this._racingCount; i++) {
+      for (let car of this._cars) {
+        // car에 score 업데이트를 위해 car에 score setter 필요??
+        car.score += this.race('-');
+        this.message += `${car.name}: ${car.score} <br />`;
+      }
+      this.message += `<br />`;
+    }
+  }
+
+  goOrStop() {
+    return Math.floor(Math.random() * 10) >= 4;
+  }
+
+  race(char) {
+    return this.goOrStop() ? char : '';
+  }
```

Comment:
아래처럼 default parameter를 활용해보셔도 좋을거 같네요~

```suggestion
  race(char = "-") {
    return this.goOrStop() ? char : '';
  }
```